src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_method_confs.cljs

```diff
@@ -36,10 +38,126 @@
     (str k "," (get the_map k) " ; ")))
 
 
+
+(react/defc ImportWorkspaceMethodsConfigurationsList
+  {:render
+   (fn [{:keys [props refs state]}]
+     [:div {}
+      (if (zero? (count (:method-confs props)))
+        [:div {:style {:textAlign "center" :backgroundColor (:background-gray style/colors)
+                       :padding "1em 0" :margin "0 4em" :borderRadius 8}}
+         "There are no method configurations to display."]
+        [table/AdvancedTable
+         (let [header (fn [children] [:div {:style {:fontWeight 600 :fontSize 13
+                                                    :padding "14px 16px"
+                                                    :borderLeft "1px solid #777777"
+                                                    :color "#fff" :backgroundColor (:header-darkgray style/colors)}}
+                                      children])
+               cell (fn [children] [:span {:style {:paddingLeft 16}} children])]
+           {:columns [{:header-component [:div {:style {:padding "13px 0 12px 12px"
+                                                        :backgroundColor (:header-darkgray style/colors)}}
+                                          [:input {:type "checkbox" :ref "allcheck"}]
+                                          ]
+                       :starting-width 42 :resizable? false
+                       :cell-renderer (fn [row-num conf]
+                                    [:div
+                                      {:style {:paddingLeft 12}} [:input {
+                                         :type "checkbox" :id (str "import_mc_checkbox_" row-num )
```

Comment:
The proper way to do this in React is by using refs.  Just use :ref instead of :id here, then retrieve it via (@refs theID) below.